imageruler/imageruler.py

```diff
@@ -756,20 +791,22 @@ def _proper_unpad(arr, kernel):
     return arr[slices]
 
 
-def _trim(arr, margin_size, pixel_size):
+def _trim(arr: np.ndarray, margin_size: Tuple, pixel_size: Tuple) -> np.ndarray:
     """
-    Obtain the trimmed array with marginal regions discarded.
+    Return the array with the boundaries removed.
```

Comment:
I am unsure whether `boundaries` here could be ambiguous. Perhaps it would be better to explicitly say `marginal regions`?